www/source/partials/docs/_reference-environment-vars.html.md.erb

```diff
@@ -7,6 +7,7 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_AUTH_TOKEN` | build system | no default | Authorization token used to perform privileged operations against the depot, e.g. uploading packages or keys.
 | `HAB_BINLINK_DIR` | build system | `/hab/bin` | Allows you to change the target directory for the symlink created when you run `hab pkg binlink`. The default value is already included in the `$PATH` variable inside the Studio. |
 | `HAB_CACHE_KEY_PATH` | build system, Supervisor | `/hab/cache/keys` if running as root; `$HOME/.hab/cache/keys` if running as non-root | Cache directory for origin signing keys |
+| `HAB_CTL_SECRET` | Supervisor | Generates a new key, if one is not present | Authentication key that is used to pass commands to a remote Supervisor via the [control gateway](/docs/internals#control-gateway). |
```

Comment:
The existing entry here seems fine. See https://www.habitat.sh/docs/reference/#environment-variables